Jaeger: Search traces (#32805)
grafana/grafana: docs/sources/datasources/jaeger.md

You can query and display traces from Jaeger via [Explore]({{< relref "../explore/_index.md" >}}).

The Jaeger query editor allows you to query by trace ID directly or use the search form to find traces. To query by trace ID, select the TraceID from the Query type selector and insert the ID into the text input.

For searching set the query type selector to Search. You can use the following fields for finding traces:

1. Service - Lists services.
1. Operation - Populated after selecting a service. Lists operations related to the selected service. Select [All] option to query all the operation.
1. Tags - Use values in the [logfmt](https://brandur.org/logfmt) format. For example `error=true db.statement="select * from User"`.

> 1. Min Duration - Setting this field should filter traces that has higher duration than the value. Possible values are `1.2s, 100ms, 500us`.

Reviewer: ```suggestion
1. Min Duration - Filter all traces with a duration higher than the set value. Possible values are `1.2s, 100ms, 500us`.
```

> 1. Max Duration - Setting this field should filter traces that duration are not higher than the value. Possible values are `1.2s, 100ms, 500us`.

Reviewer: ```suggestion
1. Max Duration - Filter all traces with a duration lower than the set value. Possible values are `1.2s, 100ms, 500us`.
```

1. Limit - Limits the number of traces returned.

## Linking Trace ID from logs

You can link to Jaeger trace from logs in Loki by configuring a derived field with internal link. See the [Derived fields]({{< relref "loki.md#derived-fields" >}}) section in the [Loki data source]({{< relref "loki.md" >}}) documentation for details.

## Configure the data source with provisioning

> You can set up the data source via configuration files with Grafana's provisioning system. Refer to [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}) for information on various settings and how it works.

Reviewer: ```suggestion
You can set up the data source via configuration files with Grafana's provisioning system. Refer to [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}) for more information on configuring various settings.
```
